.github/workflows/vulncheck.yml

```diff
@@ -0,0 +1,17 @@
+name: VulnCheck
+on:
+  pull_request:
+    branches:
+    - master
+  push:
+    branches:
+    - master
+jobs:
+  vulncheck:
+    name: Analysis
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v3
+    - name: Check for vulnerabilities
+      uses: kmulvey/govulncheck-action@main
```

Comment:
@aead Should we pin the version?

I don't know what it could mess up, but I don't know @kmulvey - and a pinned version would not make it possible to sneak in changes - even if very unlikely.

Reply:
i just tagged and published v1.0.0  https://github.com/kmulvey/govulncheck-action/releases/tag/v1.0.0

hope that helps